### ISSUE TITLE
Map - viewport is slightly too high, creates unnecessary vertical scroll pane

### ISSUE THREAD

**Issue by Nuanda** [Nuanda/smogmapper#10]

**Comment by kammerer** [comment]
It seems to me that the scrollbar is shown because there is a lot of sidebar content. Perhaps only this content should be scrolled?

**Comment by Nuanda** [comment]
I don't think so. Even removing the does not affect this. I suspects this has something to do with displacing the #main container somewhat lower, to make room for the top navbar, and the height might be assigned incorrectly. Or something like that.
